### PR TITLE
Fixes and edits : Day 9 part 1

### PR DESCRIPTION
**Pull request by orian34**
3 missing sprites that will be added later when available.

### CODE REVIEW

**Comment by enumag** [comment]
I've merged what I could but the rest of the pull requests will sadly need some reverting to use capital letters. If you're already fed up with Oni I can do it myself but not this week. Himatsubushi and Meakashi pre-releases have a bit more priority now.

File: Update/onik_009.txt [hunk 1]
```diff
@@ -2659,12 +2674,9 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { ClearMessage(); } else { OutputLineAll(NULL, "\n", Line_ContinueAfterTyping); }
 
 
-	DisableWindow();
-	ModDrawCharacter(1, 2, "sprite/normal/re2b_bikkuri_b1_", "2", -160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 0, 200, TRUE );
-
 //「え？!w400　えぇ？！!w300　なに？!w200　なに？！＠
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
-	ModPlayVoiceLS(3, 2, "s19/02/500200839", 256, TRUE);
+	ModPlayVoiceLS(3, 4, "s19/02/500200839", 256, TRUE);
```
Comment (enumag) [inline Update/onik_009.txt]: Are you sure about this? If I remember correctly then the second parameter is the number of the character talking and Rena has number 2. Generally it should match the number in the voice link s19/**02**/500200839.

Comment (enumag) [reply Update/onik_009.txt]: `ModDrawCharacter` also has this number as second parameter. This is used to match which voice animates which sprite on the screen.

**Comment by orian34** [comment]
I unswapped the numbers

**Pull request merged**